environments/ithc/service-core-compute-ithc-internal.yml

```diff
@@ -12,41 +12,5 @@ vnet_links:
     vnet_id: "/subscriptions/ed302caf-ec27-4c64-a05e-85731c3ce90e/resourceGroups/reformMgmtCoreRG/providers/Microsoft.Network/virtualNetworks/reformMgmtCoreVNet"
   - link_name: "bastion-prod-vnet"
     vnet_id: "/subscriptions/2b1afc19-5ca9-4796-a56f-574a58670244/resourceGroups/bastion-prod-rg/providers/Microsoft.Network/virtualNetworks/bastion-prod-vnet"
-A:
-  - name: camunda-api-ithc
-    record:
-    - 10.10.40.121
-    ttl: 300
-  - name: ia-home-office-integration-api-ithc
-    record:
-    - 10.10.37.250
-    ttl: 300
-  - name: ia-home-office-mock-api-ithc
-    record:
-    - 10.10.37.250
-    ttl: 300
-  - name: finrem-emca-ithc
-    record:
-    - 10.10.37.250
-    ttl: 300
-  - name: finrem-ns-ithc
-    record:
-    - 10.10.37.250
-    ttl: 300
-  - name: finrem-ps-ithc
-    record:
-    - 10.10.37.250
-    ttl: 300
-  - name: finrem-cos-ithc
-    record:
-    - 10.10.37.250
-    ttl: 300
-  - name: finrem-dgcs-ithc
-    record:
-    - 10.10.37.250
-    ttl: 300
-  - name: ccd-data-0
```

Comment:
Can we confirm if this is needed? `ccd-data-0` I dont think it is, looks to be from consul
https://github.com/search?q=org%3Ahmcts+ccd-data-0&type=code

Reply:
Can't see anywhere else where it's being used and yes it's consul related.  May be better to leave alone for now and do a separate consul records cleanup?

Reply:
[Record added back](https://dev.azure.com/hmcts/CNP/_build/results?buildId=147135&view=logs&j=3b6b9102-0389-5812-f6cc-5470a3c8128d&t=a8432ae7-1cee-57c7-171e-10e43cf7923c&l=34)